scripts/generate_pipeline_files.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+
+import json
+import os
+import pathlib
+
+import click
+from ruamel import yaml
+from ruamel.yaml.comments import CommentedMap
+
+yaml = yaml.YAML()
+yaml.representer.ignore_aliases = lambda *data: True
+
+CURRENT_PATH = pathlib.Path(__file__).resolve().parent
+PROJECT_ROOT = CURRENT_PATH.parent
+DATASETS_PATH = PROJECT_ROOT / "datasets"
+AIRFLOW_TEMPLATES_PATH = PROJECT_ROOT / "templates" / "airflow"
+
+license_header = (
+    pathlib.Path(AIRFLOW_TEMPLATES_PATH / "license_header.py.jinja2").read_text() + "\n"
+)
+
+
+@click.command()
+@click.option(
+    "--dataset_id",
+    prompt="Your dataset name",
+    required=True,
+    type=str,
+    help="Dataset Name or Category that your pipeline belongs to",
+)
+@click.option(
+    "--pipeline_id",
+    "-p",
+    prompt="Your pipeline name",
+    required=True,
+    type=str,
+    help="The name of your pipeline",
+)
+def create_pipeline(dataset_id: str, pipeline_id: str):
+    new_pipeline_path = f"{DATASETS_PATH}/{dataset_id}/pipelines/{pipeline_id}/"
+    dir = os.path.dirname(new_pipeline_path)
+    if not os.path.exists(dir):
+        os.makedirs(dir)
+        click.echo(
+            f"\n{DATASETS_PATH}/{dataset_id}/pipelines/{pipeline_id} has been created\n"
+        )
+    else:
+        click.echo(f"\n{new_pipeline_path} already exists\n")
+    create_dataset_yaml(dataset_id)
+    create_pipeline_yaml(dir)
+
+
+def create_dataset_yaml(dataset_id: str):
+    output = {}
+    sample_yaml = yaml.load((PROJECT_ROOT / "samples" / "dataset.yaml").read_text())
```

Comment:
The term `sample_` might be outside of the context here. Let's use `template` to mean this is a dict that needs to have its values set.